VocaDbModel/Domain/OptionalGeoPoint.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+
+namespace VocaDb.Model.Domain {
+
+	public interface IOptionalGeoPoint {
+
+		double? Latitude { get; }
+
+		double? Longitude { get; }
+
+	}
+
+	public class OptionalGeoPoint : IOptionalGeoPoint {

```

Comment:
Maybe add a small comment here. If Latitude/Longitude is specified, the other one needs to be specified too, but it's ok for both to be empty.